jobs/cache_maintenance/tests/utils.py

```diff
@@ -56,25 +56,24 @@ def update_repo_settings(
             <Tip>
             Version added: 0.5
             </Tip>
-        private (`bool`, *optional*, defaults to `None`):
+        private (`bool`, *optional*):
             Whether the repo should be private.
-        gated (`str`, *optional*, defaults to `None`):
+        gated (`str`, *optional*):
             Whether the repo should request user access.
             Possible values are 'auto' and 'manual'
         token (`str`, *optional*):
             An authentication token (See https://huggingface.co/settings/token)
         repo_type (`str`, *optional*):
             Set to `"dataset"` or `"space"` if uploading to a dataset or
-            space, `None` or `"model"` if uploading to a model. Default is
-            `None`.
-    Returns:
-        The HTTP response in json.
-    <Tip>
-    Raises the following errors:
-        - [`~huggingface_hub.utils.RepositoryNotFoundError`]
+            space, `None` or `"model"` if uploading to a model.
+
+    Raises:
+        [~`huggingface_hub.utils.RepositoryNotFoundError`]
```

Comment:
I think it's the other way, no?

```suggestion
        [`~huggingface_hub.utils.RepositoryNotFoundError`]
```

https://www.notion.so/huggingface2/Documentation-926791574d274a5194cc83699c4a238e

Reply:
sorry, https://www.notion.so/huggingface2/Documentation-926791574d274a5194cc83699c4a238e is not source, it's https://github.com/huggingface/doc-builder